benchmark/src/runner.rs

```diff
@@ -208,7 +202,7 @@ impl CommandRunner {
             let tx_clone = tx.clone();
             let handle = tokio::spawn(async move {
                 let mut key = vec![0u8; key_size];
-                let _ = c.wait().await;
+                let _: tokio::sync::BarrierWaitResult = c.wait().await;
```

Comment:
We can modify this line of code as `_ = c.wait().await`. It's more concise.

Reply:
done